dask/array/tests/test_cupy.py

```diff
@@ -37,8 +37,8 @@
     pytest.param(
         lambda x: x.mean(),
         marks=pytest.mark.skipif(
-            not IS_NEP18_ACTIVE or cupy_version < parse_version("6.4.0"),
-            reason="NEP-18 support is not available in NumPy or CuPy older than "
+            cupy_version < parse_version("6.4.0"),
+            reason="NEP-18 support is not available in CuPy older than "
             "6.4.0 (requires https://github.com/cupy/cupy/pull/2418)",
```

Comment:
```suggestion
            reason="CuPy older than 6.4.0 (requires "
            "https://github.com/cupy/cupy/pull/2418)",
```

Reply:
How about 

```suggestion
            reason="Requires CuPy 6.4.0+"
            "(with https://github.com/cupy/cupy/pull/2418)",
```

Reply:
Sure, that's fine by me too.

Reply:
Ok these are all now updated. Please take another look :)